platform/broadcom/rules.dep

```diff
@@ -0,0 +1,25 @@
+#DPKG FRK
+include $(PLATFORM_PATH)/sai-modules.dep
+include $(PLATFORM_PATH)/sai.dep
+include $(PLATFORM_PATH)/platform-modules-s6000.dep
+include $(PLATFORM_PATH)/platform-modules-dell.dep
+include $(PLATFORM_PATH)/platform-modules-arista.dep
+include $(PLATFORM_PATH)/platform-modules-ingrasys.dep
+include $(PLATFORM_PATH)/platform-modules-accton.dep
+include $(PLATFORM_PATH)/platform-modules-alphanetworks.dep
+include $(PLATFORM_PATH)/platform-modules-inventec.dep
+include $(PLATFORM_PATH)/platform-modules-cel.dep
+include $(PLATFORM_PATH)/platform-modules-delta.dep
+include $(PLATFORM_PATH)/platform-modules-quanta.dep
+#include $(PLATFORM_PATH)/platform-modules-mitac.dep
```

Comment:
not sure why mitac dep is commented out here.

Reply:
The mitac.mk file is commented (#) on the platform/broadcom/rules.mk file. Hence excluded in the dep list as well.